### ISSUE TITLE
Admin UI: Show machine tags

### ISSUE THREAD

**Issue by dkerwin** [tailscale/tailscale#576]
**Describe the solution you'd like**

I'm using tags on my servers and it would be great the see (and filter based on) the machine tags.

**Comment by llewxam-kache** [comment]
+1 to this, even if you can only see them if you're on the local host, but ideally an admin would be able to see what tags each machine is advertising

**Comment by rosszurowski** [comment]
Yep, totally agree! ACL groups are probably also relevant to see.

We're working on a feature right now that'll require some new UI on the machines page, and I'd like to roll this into those changes.

**Comment by bradfitz** [comment]
I was also going to add this to the `tailscale status {,--json}` CLI output, but I realized the coordination server never even sends this information back down.

@crawshaw, should I add it?

**Comment by rosszurowski** [comment]
Forgot to mention here, but tags are now visible in the UI as of March and Tailscale v1.6. More details here: https://tailscale.com/blog/2021-03-newsletter/#admin-panel-updates

**Comment by DentonGentry** [comment]
https://github.com/tailscale/tailscale/issues/1360 tracks adding tags to the API. We have two engineers working on authkeys and tags features, I'd expect the API to be addressed as part of that effort.

Another part of that effort is to address https://github.com/tailscale/tailscale/issues/1369, allowing authkeys to carry a set of tags to be applied to nodes it creates. As such, a node can have tags applied which it wasn't involved in adding. Though we could communicate the tags to the node to appear in `tailscale status --json`, I think we'd be better off pointing to the API for that.

Barring further comment, we'll expect to close this bug during the next bug scrub and handle the next step on https://github.com/tailscale/tailscale/issues/1360
